sdk/resourcemanager/azure-resourcemanager-containerservice/src/test/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterAgentPoolImplTests.java

```diff
@@ -61,7 +61,9 @@ void testGetAgentPoolInner() throws Exception {
             String name = method.getName();
             if (mockValues.containsKey(name)) {
                 Object value = method.invoke(agentPoolInner);
-                Assertions.assertEquals(mockValues.get(name), value, String.format("Field %s mismatch", name));
+                if (value != null) {
+                    Assertions.assertEquals(mockValues.get(name), value, String.format("Field %s mismatch", name));
+                }
```

Comment:
revert this, it is a good test

Reply:
You need to add `messageOfTheDay` to https://github.com/Azure/azure-sdk-for-java/blob/8b93c3db2501be9f540862f72c51d931d0096fd4/sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterAgentPoolImpl.java#L277

Reply:
fix by [4ff06b1](https://github.com/Azure/azure-sdk-for-java/pull/44056/commits/4ff06b19a936d1aee96b80badfdce758dc679039) and [7f3536c](https://github.com/Azure/azure-sdk-for-java/pull/44056/commits/7f3536c42c01d8f5597d0d092677a4c719801154)